ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobMetadata.java

```diff
@@ -345,6 +353,46 @@ public CloudBlobMetadata setEncryptedSize(long encryptedSize) {
     return this;
   }
 
+  /**
+   * @return the last update time of the blob.
+   */
+  public long getLastUpdateTime() {
+    return lastUpdateTime;
+  }
+
+  /**
+   * Sets the last update time of the blob.
+   * @param lastUpdateTime last update time.
+   */
+  public void setLastUpdateTime(long lastUpdateTime) {
+    this.lastUpdateTime = lastUpdateTime;
+  }
+
+  /**
+   * Utility to cap specified {@link CloudBlobMetadata} list by specified size of its blobs.
+   * Always returns at least one metadata object irrespective of size.
+   * @param originalList List of {@link CloudBlobMetadata}.
+   * @param size size total size of metadata's blobs.
+   * @return {@link List} of {@link CloudBlobMetadata} capped by size.
+   */
+  public static List<CloudBlobMetadata> capMetadataListBySize(List<CloudBlobMetadata> originalList, long size) {
```

Comment:
I'd prefer to keep CloudBlobMetadata a pojo class and move this method to a utility class that both prod and test code can call.

Reply:
I couldn't find a better place for this code. I could maybe create a CloudUtil class, but it will have only one method for now.

Reply:
As discussed we will leave it as is for now.